src/olympia/pages/templates/pages/faq.html

```diff
@@ -181,25 +157,23 @@ <h3 id="gallery">{{ _('Add-on Gallery') }}</h3>
          With thousands of add-ons available, there's something for everyone.
          But if you're looking for a particular add-on and can't find it, you
          might try searching on other sites or posting about it in our
-         <a href="{{ forum_url }}">Add-ons </a>forum.{% endtrans %}</dd>
+         <a href="{{ forum_url }}">Add-ons forum</a>.{% endtrans %}</dd>
 
     <dt id="experimental">{{ _('What does it mean if an add-on is "experimental"?') }}</dt>
-    <dd>{% trans url="https://addons.mozilla.org/developers/docs/policies/reviews" %}
+    <dd>{% trans url="https://developer.mozilla.org/Add-ons/AMO/Policy/Reviews" %}
```

Comment:
Can we add the language slug here? Otherwise, non en-US browsers will be redirected to that page in their locale which might or might not exist.